src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-lentgh--februaryOdd calendar--start-day--sunday">
```

Comment:
```suggestion
    <div class="calendar calendar--month-lentgh--29 calendar--start-day--sunday">
```
Сюда сразу число приходит, не название месяца